<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderWindow</class>
 <widget class="QDialog" name="OrderWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>475</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>950</width>
    <height>475</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>950</width>
    <height>475</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Замовлення</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>621</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>10</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Замовлення</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Контрагент</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_add_new_contragent">
   <property name="geometry">
    <rect>
     <x>349</x>
     <y>50</y>
     <width>321</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Додати нового контрагента</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_sale">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Продаж</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_2_buy">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Покупка</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Список товарів</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Кількість</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Ціна за одницю</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Валюта</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2_add_next_commodity">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>220</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Додати наступний товар</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_contragent" native="true">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>90</y>
     <width>321</width>
     <height>220</height>
    </rect>
   </property>
   <widget class="QRadioButton" name="radioButton_3_person_entity">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Фіз_особа</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_4_legal_entity">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Юр_особа</string>
    </property>
   </widget>
   <widget class="QWidget" name="legal_entity" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>281</width>
      <height>150</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>101</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Назва</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>101</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>ЄДРПОУ*</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>111</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Ідент. код*</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>101</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Телефон*</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_5_legal_entity_name">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>150</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_6_legal_entity_id_code">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>150</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_7_legal_entity_edrpou">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>150</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_8_legal_entity_telephone">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>150</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="person_entity" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>281</width>
      <height>175</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_20_person_entity_name">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>101</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>ПІБ</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_21_person_entity_date_birth">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>101</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Дата нар.*</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_22_person_entity_telephone">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>111</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Телефон*</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_9">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>150</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_10">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>150</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QDateEdit" name="lineEdit_11">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>150</width>
       <height>25</height>
      </rect>
     </property>
     <property name="displayFormat">
      <string>dd/MM/yyyy</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_add_new_contragent_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>170</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Додати</string>
    </property>
   </widget>
   <zorder>radioButton_3_person_entity</zorder>
   <zorder>radioButton_4_legal_entity</zorder>
   <zorder>pushButton_add_new_contragent_2</zorder>
   <zorder>legal_entity</zorder>
   <zorder>person_entity</zorder>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2_quantity">
   <property name="geometry">
    <rect>
     <x>141</x>
     <y>210</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3_price">
   <property name="geometry">
    <rect>
     <x>141</x>
     <y>240</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Назва товару</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_4_currency">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>141</x>
     <y>270</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>141</x>
     <y>300</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>dd/MM/yyyy</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Дата замовлення</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_measurement_units">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>210</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="ErrorLabel_Order">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>300</y>
     <width>341</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="locale">
    <locale language="Ukrainian" country="Ukraine"/>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_name_commodity">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>180</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_contragent">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget_new_order" native="true">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>40</y>
     <width>265</width>
     <height>391</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_widget_new_order">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>181</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Нове замовлення</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_new_order">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>265</width>
      <height>361</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>130</number>
    </attribute>
    <column>
     <property name="text">
      <string>Назва товару</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Кількість</string>
     </property>
    </column>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>OrderWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>OrderWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
